blackbox_repository/utils.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import List, Optional
+import numpy as np
+
+from blackbox_repository.blackbox import Blackbox
+
+
+def metrics_for_configuration(
+        blackbox: Blackbox, config: dict, resource_attr: str,
+        fidelities: Optional[np.array] = None) -> List[dict]:
+    """
+    Returns all results for configuration `config` at fidelities in
+    `fidelities`.
+
+    :param blackbox: Blackbox
+    :param config: Configuration
+    :param resource_attr: Name of resource attribute
+    :param fidelities: Defaults to all fidelity values
```

Comment:
The intent of this one is not very clear, more than default this is more of a filter (only fidelities that are in this vector are returned). Could you update the comment to better reflect that?

Reply:
OK